docker-compose.prod.yml

```diff
@@ -1,4 +1,7 @@
 version: "3.8"
+networks:
+  proxy:
```

Comment:
Maybe we can remove the `frontend` network.

Reply:
Yes definitely. I didn't want to remove anything but should be the same